vllm/model_executor/layers/quantization/modelopt.py

```diff
@@ -553,6 +553,19 @@ def apply(
                     expert_map=expert_map,
                     apply_router_weight_on_input=apply_router_weight_on_input,
                 )
+        elif self.fused_experts is not None:
+            return self.fused_experts(
+                x,
+                layer.w13_weight,
+                layer.w2_weight,
+                topk_weights,
+                topk_ids,
+                inplace=False,
+                activation=activation,
+                global_num_experts=global_num_experts,
+                expert_map=expert_map,
+                apply_router_weight_on_input=apply_router_weight_on_input,
+            )
```

Comment:
Could we similarly use the pattern like https://github.com/vllm-project/vllm/pull/23294
So no too much code duplicated
```py
            if self.fused_experts is not None:
                return self.fused_experts(**common_kwargs)
            else:
                from vllm.model_executor.layers.fused_moe import fused_experts
                return fused_experts(
                    **common_kwargs,
                    use_fp8_w8a8=True,
                    block_shape=self.quant_config.weight_block_size,
                    allow_deep_gemm=self.allow_deep_gemm,
                    allow_cutlass_block_scaled_grouped_gemm=(
                        self.allow_cutlass_block_scaled_grouped_gemm),
                )
```

Reply:
Can we not do this?  I'm just going to have to undo it in https://github.com/vllm-project/vllm/pull/22537

Reply:
I'm not totally against this.  I'm mostly just wanting to reduce merge conflicts/potential bugs later on.  I feel like it's really easy to forget/miss arguments in dicts, especially if they are defaulted.

Reply:
Either choice is ok with me.